beaker.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |s|
   # Run time dependencies
   s.add_runtime_dependency 'minitest', '~> 5.4'
   s.add_runtime_dependency 'minitar', '~> 0.5.4'
-  s.add_runtime_dependency 'json', '~> 1.8'
+  s.add_runtime_dependency 'json', '~> 1.4'
```

Comment:
It would be nice if we could drop it like rails did. It puts us closer to allowing ruby to just use the builtin version. For example, [`multi_json.gemspec`](https://github.com/intridea/multi_json/blob/master/multi_json.gemspec) doesn't list `json` as a dependency.

If we drop it and some transitive dependency has an explicit `json` dependency (with or without a version), then I think we're back to where we are now -- bundler will try to download the latest json gem that satisfies the constraints.

That could result in us getting a 2.x `json` version. Is that bad? Beaker#master requires ruby >= 2.2.5 and json 2 requires ruby > 2, but beyond that I'm not sure what breakage could result from json 2.